mockative/build.gradle.kts

```diff
@@ -23,6 +23,7 @@ kotlin {
 }
 
 afterEvaluate {
+    println("after evaluate")
```

Comment:
Is this `println` intentionally left in there?

Reply:
Yeesh . . . no. I thought I deleted the println statements I added. I was having some trouble with the build and had put a bunch of printlns in before I realized it was a two-stage build where some code got copied in the first stage (it helps to read the instructions).